cli/tests/data/universe-v3-stub.json

```diff
@@ -26,14 +26,14 @@
           "containerPort": 80
         }
       ],
-      "image": "mesosphere/universe-server:20161006T215648Z-cli-test-v3-941b817f88",
+      "image": "mesosphere/universe-server:20170326T170201Z-version-3.x-34bf4c095b",
       "network": "BRIDGE"
     },
     "type": "DOCKER"
   },
   "requirePorts": true,
   "mem": 128,
-  "cpus": 0.25,
+  "cpus": 0.1,
   "instances": 1,
   "id": "/universe"
 }
```

Comment:
This is one of the important changes. Update the Universe repo to the latest set of packages.